src/App.tsx

```diff
@@ -4,15 +1,35 @@
 import './App.scss';
+import { NavLink, Outlet } from 'react-router-dom';
+import classNames from 'classnames';
 
 export const App = () => {
+  const getActiveLink = ({ isActive }: { isActive: boolean }) =>
+    classNames('navbar-item', { 'has-background-grey-lighter': isActive });
+
   return (
     <div data-cy="app">
-      <Navbar />
+      <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink className={getActiveLink} to="/">
+          Home
+        </NavLink>
+
+        <NavLink className={getActiveLink} to="/people">
```

Comment:
Similar to the previous `NavLink`, ensure that the `to` prop value `/people` matches a valid route in your router configuration. This will prevent broken links or navigation errors.